pkg/hook/exechook.go

```diff
@@ -59,7 +59,7 @@ func (h *Exechook) Name() string {
 }
 
 // Do runs exechook.command, implements Hook.Do
-func (h *Exechook) Do(ctx context.Context, hash string) error {
```

Comment:
Huh, I could swear we were setting the has in an env var.  I can add that later

Reply:
I'm still not sure we should do this, but if you rebase you will now find that we set $GITSYNC_HASH :)

Reply:
So *if* we were to proceed with this proposal and add a header in HTTP, wew should add the same in exec.